### ISSUE TITLE
Transferring jupyter ydoc (and pycrdt?)

### ISSUE THREAD

**Issue by fcollonval** [jupyterlab/frontends-team-compass#257]
### Problem

Collaboration features are provided by the extension https://github.com/jupyterlab/jupyter-collaboration
But some key components (ydoc, pycrdt and pycrdt websocket) are not in Jupyterlab org but in jupyter server. This results in maintainers rights discrepancies between repo and increase rights management fatigue. 

What do you think about moving all collaboration related in the same org? Which one would be best; jupyterlab or jupyter-server?

**Comment by echarles** [comment]
fyi we had done some work in https://github.com/jupyter-rtc (which could be renamed to https://github.com/jupyter-collaboration). Having such a common org for both client and server side code would remove those difficulties).

**Comment by davidbrochart** [comment]
It's true that we need to reduce maintenance pain, but we also need to put the repositories in the right place. If we don't create a new https://github.com/jupyter-collaboration organization, we could:
- leave [pycrdt](https://github.com/jupyter-server/pycrdt), [pycrdt-websocket](https://github.com/jupyter-server/pycrdt-websocket) and [jupyter_ydoc](https://github.com/jupyter-server/jupyter_ydoc) in `jupyter-server`, and move [jupyter_server_ydoc](https://github.com/jupyterlab/jupyter-collaboration/tree/main/projects/jupyter-server-ydoc) in there.
- leave [jupyter-collaboration-ui](https://github.com/jupyterlab/jupyter-collaboration/tree/main/projects/jupyter-collaboration-ui), [jupyter-docprovider](https://github.com/jupyterlab/jupyter-collaboration/tree/main/projects/jupyter-docprovider) and [jupyter-collaboration](https://github.com/jupyterlab/jupyter-collaboration/tree/main/projects/jupyter-collaboration) in `jupyterlab`.

So basically just move [jupyter_server_ydoc](https://github.com/jupyterlab/jupyter-collaboration/tree/main/projects/jupyter-server-ydoc) in https://github.com/jupyter-server, which shouldn't live in https://github.com/jupyterlab (see https://github.com/jupyterlab/jupyter-collaboration/issues/391#issuecomment-2446316936).

**Comment by jtpio** [comment]
For reference there are some points for *not* creating additional GitHub organizations: https://github.com/jupyter-governance/ec-team-compass/issues/12
